### PR TITLE
test on old node.js

### PR DESCRIPTION
**Pull request by fisker**
test on old node.js

### CODE REVIEW

**Comment by brodybits** [comment]
I think you need to do `yarn install --ignore-engines` for the dev test to succeed on Node.js 6, like I did in d2a3ad309168e58c1d4426c73d8e7660e6498bdc.

It also looks like this proposal was based on some changes that I had proposed in PR #6232. I would appreciate credit in a `Co-authored-by` comment in the git commit message (see [[1]](https://help.github.com/en/articles/creating-a-commit-with-multiple-authors)).

[1] <https://help.github.com/en/articles/creating-a-commit-with-multiple-authors>

File: azure-pipelines.yml [hunk 1]
```diff
@@ -38,11 +38,19 @@ jobs:
       - template: .azure-pipelines/jobs/dev-test.yml
 
   - job: Dev_Test_Linux
-    displayName: Dev Test on Linux Node v10
+    displayName: Dev Test on Linux Node.js
```
Comment (brodybits) [inline azure-pipelines.yml]:
```suggestion
    displayName: Dev Test on Linux
```

Comment (fisker) [reply azure-pipelines.yml]: I know you did last time, I'm thinking another way to test, maybe later

Comment (fisker) [reply azure-pipelines.yml]: we can't do `yarn install --ignore-engines`, this requires people install with this argument, no one wan't that

**Comment by fisker** [comment]
will try on fork

**Pull request closed without merging**